src/main/java/org/syftkog/web/test/framework/EnvironmentType.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2014 BenjaminLimb.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.syftkog.web.test.framework;
+
+/**
+ *
+ * @author BenjaminLimb
+ */
+public enum EnvironmentType {

```

Comment:
As this is an open source project you might want to make this an interface/class instead of an enum as other organizations may have different environment needs. You could still provide this enum and have it implement the interface if you wanted a set of predetermined ones to be available still allowing the environment to be anything (see Effective Java 2nd Edition, Item 34: Emulate extensible enums with interfaces).